Агаев Савелий, 204

BeginTheorem opened the pull request:
Дз по numpy. Версия №1

BeginTheorem committed:
Create untitled0.py

vongostev commented on numpy_1/untitled0.py:
```diff
@@ -0,0 +1,82 @@
+def function_to_test_poi(l, N):
+    try:
+        poi(l, N)
+    except ValueError:
```
Лучше не писать исключения без описания, потому что будет непонятно, что ожидалось, и что пошло не так.

vongostev reviewed (approved):
Все отлично, но учтите мое замечание

BeginTheorem committed:
Create untitled1.py

BeginTheorem commented:
Дз по numpy. Версия №2. Файл перед комментарием. 

Прошу простить, с переименованием отправляемых файлов возникла сложность.

BeginTheorem committed:
Переименован. Отформатирован.

vongostev commented:
Супер!

BeginTheorem committed:
Задача N-тел. Попытка 1.

Здесь то, чего получилось добиться за три недели.
Некоторые элементы кода, вероятно, могут быть оптимизированы. Графика нуждается в  некоторой отладке.
В целом,  реализация отражает то, что получилось на данный момент регуляризовать из исходного задания, соображений математики и здравого смысла.

vongostev commented:
Реализация огонь! Но немного изощренная. Логирование огонь! Названия планет огонь!
По поводу конфигов -- создавать такие штуки в csv не очень хорошо, вот тут Вам аж 2 конфига понадобилось. Для конфигов есть свои форматы, советую посмотреть jso, ini и yaml, это стандарты.
Зачет
